profile/README.md

```diff
@@ -0,0 +1,24 @@
+<h2 align="center">JSON Schema is a declarative language that allows you to annotate and validate JSON documents</h2>
+
+[![Slack](https://img.shields.io/static/v1?label=Slack&message=@json-schema&color=yellow)](https://asyncapi.slack.com)
+[![Twitter](https://img.shields.io/static/v1?label=Twitter&message=@jsonschema&color=9cf)](https://twitter.com/jsonschema)
+[![LinkedIn](https://img.shields.io/static/v1?label=LinkedIn&message=@jsonschema&color=lightgray)](https://www.linkedin.com/company/jsonschema)
+[![YouTube](https://img.shields.io/static/v1?label=Youtube&message=@JSONSchemaOrgOfficial&color=red)](https://www.youtube.com/@JSONSchemaOrgOfficial)
+[![Google Groups](https://img.shields.io/static/v1?label=Google&nbsp;Groups&message=@json-schema&color==blueviolet)](https://groups.google.com/g/json-schema)
+
+### 👋 Welcome
+
+Welcome to JSON Schema, a vocabulary that allows you to validate, annotate, and manipulate JSON documents.
+
+JSON Schema is part of [the OpenJS Foundation](https://openjsf.org/blog/2022/01/31/json-schema-joins-openjs-foundation/).
+
+### 🌎 Join JSON Schema Community
+To get involved with our Community, please make sure you are familiar with the project's [Code of Conduct](https://github.com/json-schema-org/.github/blob/main/CODE_OF_CONDUCT.md).
+
+- **Join our [Slack workspace](https://json-schema.org/slack)**: The best place to ask questions, learn, get help, or discuss all things JSON Schema.
+- **Join our public meetings**: We hold monthly [Office Hours](https://github.com/orgs/json-schema-org/discussions/34) and weekly [Open Community Working Meetings](https://github.com/orgs/json-schema-org/discussions/35). Please join!
+- **JSON Schema [YouTube Channel](https://www.youtube.com/watch?v=48S8-GwRh-g&list=PLHVhS4Tj1YZPYt6sMkvf4nW8zKvZExVA4)**: To find the recordings of our public community meetings and JSON Schema learning resources.
```

Comment:
Minor, but the first two bullets here have verbs, and this one doesn't, i.e.:

> Join our

> Join our

but this one just says the noun (the youtube channel).

Perhaps instead:

```suggestion
- **Follow our [YouTube Channel](https://www.youtube.com/watch?v=48S8-GwRh-g&list=PLHVhS4Tj1YZPYt6sMkvf4nW8zKvZExVA4)**: To find the recordings of our public community meetings and JSON Schema learning resources.
```